src/RandomGenerator.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2015, DFKI GmbH Robotics Innovation Center
+ *
+ *  This file is part of the MARS simulation framework.
```

Comment:
whoops. I just copied the header from DummyItem. Is there a license header for envire?